ops/ccs-ops-misc/load_test/common/url_path.py

```diff
@@ -0,0 +1,13 @@
+import time
+from typing import Dict
+from urllib.parse import urlencode
+
+'''
+Creates a query path from a base path (i.e. /v2/fhir/Coverage) and a dictionary of query parameters
+'''
+def create_url_path(path: str, query_params: Dict[str, str] = {}) -> str:
```

Comment:
Huh didnt know you could specify types for params in python, I like it.

Reply:
Yeah, I didn't either until working on this. Albeit I think they're just type hints opposed to actual compiler enforced types, so I think a caller could still pass a non-`str` type for `path` or a `list` for `query_params` and you there'd be no complaints from Python until runtime.